openedx/adg/lms/applications/models.py

```diff
@@ -36,6 +45,12 @@ class ApplicationHub(TimeStampedModel):
         default=False, verbose_name=_('Written Application Submitted'),
     )
     submission_date = models.DateField(null=True, blank=True, verbose_name=_('Submission Date'), )
+    course_status = models.CharField(
+        verbose_name=_('Omni Course Status'), choices=STATUS_CHOICES, max_length=15, blank=True, null=True,
+    )
+    bu_course_status = models.CharField(
+        verbose_name=_('Business Line Course Status'), choices=STATUS_CHOICES, max_length=15, blank=True, null=True,
```

Comment:
We can make the verbose name more descriptive, like "Business Unit Prerequisite Courses Status"